pkg/image/filefmt.go

```diff
@@ -54,13 +54,14 @@ var knownHeaders = Headers{
 	},
 	"vdi": Header{
 		Format:      "vdi",
-		magicNumber: []byte("<<< Oracle VM"),
+		magicNumber: []byte{0x7F, 0x10, 0xDA, 0xBE},
+		mgOffset:    0x40,
```

Comment:
these fixes should probably be backported

Reply:
Upon further though, probably not necessary because `qemu-img convert` is still called